Source/DafnyLanguageServer.Test/GutterStatus/LinearVerificationGutterStatusTester.cs

```diff
@@ -114,7 +114,8 @@ public abstract class LinearVerificationGutterStatusTester : ClientBasedLanguage
 
   protected async Task<List<LineVerificationStatus[]>> GetAllLineVerificationStatuses(
       TextDocumentItem documentItem,
-      TestNotificationReceiver<VerificationStatusGutter> verificationStatusGutterReceiver
+      TestNotificationReceiver<VerificationStatusGutter> verificationStatusGutterReceiver,
+      bool intermediates = true
```

Comment:
This mechanism was introduced in the yet unmerged PR
https://github.com/dafny-lang/dafny/pull/3116
and enables only testing the final gutter icons, not the intermediates, so that it's both easier and supports concurrent verification.